LBBT_Tests/UnitTest1.cs

```diff
@@ -13,14 +10,7 @@ public class Tests
     [TestCase(145050, 1)]
     public void Test_0pcRate(int price, int tax)
     {
-        // Arrange
-        LBBTCalculator _aTaxCalculator = new();
-
-        // Act
-        double _actual = _aTaxCalculator.CalculateTotalTax(price);
-
-        // Assert
-        Assert.That(_actual, Is.EqualTo(tax));
+        DoTest(price, tax, nameof(Test_0pcRate));
```

Comment:
Put in a common method but passing in the name of this method as it can be helpful to have that in assertion messages once it gets more complex